thirdparty/mupdf/CMakeLists.txt

```diff
@@ -136,6 +136,9 @@ set(PATCH_CMD9 "${KO_PATCH} ${CMAKE_CURRENT_SOURCE_DIR}/webp-upstream-697749.pat
 # add support for cbz chapters
 set(PATCH_CMD10 "${KO_PATCH} ${CMAKE_CURRENT_SOURCE_DIR}/mupdf_cbz_chapter_support.patch")
 
+# don't try link with libraries when building generator helpers…
+set(PATCH_CMD10 "${KO_PATCH} ${CMAKE_CURRENT_SOURCE_DIR}/fix_link_cmd.patch")
```

Comment:
You probably meant `PATCH_CMD11` on line 140, that you use below.

Reply:
Damn... Good catch.